examples/fileio_features_xc/README.rst

```diff
@@ -0,0 +1,33 @@
+example: fileio_features_xc
+===========================
+
+This example show the basics functionality of xscope fileio using ``XC`` as the programming language. 
+
+Build example (using xcommon_cmake)
+-----------------------------------
+
+Run the following command from the current directory: 
+
+.. code-block:: console
+
+  cmake -G "Unix Makefiles" -B build
+  xmake -C build
+
+Running example
+---------------
+
+.. warning::
+
+  Make sure ``xscope_fileio`` host application is installed.
+  Make sure ``XCOMMON_CMAKE_PATH`` environment variable is set to the directory where ``xcommon_cmake`` is located.
```

Comment:
Should this warning be in the build section above because you don't need this environment variable to run, it's only needed to build. This also applies to the other readmes.

Reply:
true, changing it

Reply:
I only meant the warning about the XCOMMON_CMAKE_PATH environment variable should be in the build section; having the xscope_fileio host application installed is more relevant in the 'Running' section.